modules/gatk4/applyvqsr/main.nf

```diff
@@ -0,0 +1,55 @@
+process GATK4_APPLYVQSR {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda (params.enable_conda ? "bioconda::gatk4=4.2.3.0" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/gatk4:4.2.3.0--hdfd78af_0' :
+        'quay.io/biocontainers/gatk4:4.2.3.0--hdfd78af_0' }"
+
+    input:
+    tuple val(meta), path(vcf), path(tbi), path(recal), path(recalidx), path(tranches)
+    path fasta
+    path fai
+    path dict
+    val allelespecific
+    val truthsensitivity
+    val mode
+
+    output:
+    tuple val(meta), path("*.vcf.gz")     , emit: vcf
+    tuple val(meta), path("*.tbi")        , emit: tbi
+    path "versions.yml"                   , emit: versions
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.suffix ? "${meta.id}${task.ext.suffix}" : "${meta.id}"
```

Comment:
there is a new prefix annotation, see here: https://github.com/nf-core/modules/blob/4e5406c221e91ede93d9d8036110da62c86cca66/modules/abacas/main.nf#L20